### ISSUE TITLE
fix links to stylesheet and images in blog

### ISSUE THREAD

**Issue by LuzianW** [caspii/kittysplit-blog#1]
some of the stylesheets and images in the blog are linked with a static url to http://kittysplit.com, which redirects to the new www.kittysplit.com, where these resources are missing.

change to relative links to fix this
```
<link href="http://kittysplit.com/stylesheets/bootstrap.css?v=1" rel="stylesheet" type="text/css">
<link href="http://kittysplit.com/stylesheets/kitty_styles.css?v=1" rel="stylesheet" type="text/css">
<img src="http://kittysplit.com/static/logos/footerIcon.png">
```

**Comment by LuzianW** [comment]
already fixed in 62dafb2bc4c8fb1b33252f1cfcef022c25801809

**Comment by caspii** [comment]
Deployed
